addons/website/views/snippets.xml

```diff
@@ -186,7 +186,7 @@
                 <div class="col-lg-12 pt16 pb16">
```

Comment:
Random comment: our commit convention for mentioning task is: task-XXX

Reply:
Duly noted